tjmonopix2/system/bdaq53.py

```diff
@@ -21,9 +21,14 @@
 
 
 class BDAQ53(Dut):
-    '''
-    Main class for BDAQ53 readout system
-    '''
+    """Main class for BDAQ53 readout system, uses chip specific configurations form the testbench.
+    Initializes different DAQ modules (e.g. TDC, TLU, NTC...).
+
+    Parameters
+    ----------
+    Dut : class
+        DAQ and testbench configurations of the DUT
+    """
```

Comment:
Basically, this controls the readout board and its firmware modules. It dos not use chip-specific configurations as the chip is a separate class.
The `Dut` class is a base class from `basil`.